spec/lib/services/account-api-service-spec.js

```diff
@@ -0,0 +1,154 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+require('../../helper');
+
+describe("Account API Service", function() {
```

Comment:
It lacks following two kinds of negative unit-test:
(1) If the `username` is not passed to some APIs (like getUser, deleteUser, modifyUser, .etc), these APIs should return a rejected promise.
(2) If the `username` is specified, but fail to find the user info from document, these API should also return a rejected promise.